### ISSUE TITLE
Suggestion : Launcher activity indicator.

### ISSUE THREAD

**Issue by Nolram12345** [btbd/mmultiplayer#23]
Like the title says, an indication that the Launcher is running. People (including me) might get a little confused when nothing happens when they launch the Launcher, so a little window saying that it is active might be very useful.

**Comment by BoffinBrain** [comment]
Don't you get the following dialog when you run the launcher?

![image](https://user-images.githubusercontent.com/13255058/90427422-6ae2aa80-e0ba-11ea-86de-4bef38ef59db.png)

**Comment by Nolram12345** [comment]
Yes, but that window vanishes once its active. In the beginning I was unsure whether it was running or it crashed.

**Comment by BoffinBrain** [comment]
Ah, I see. All the time I've been using it, I've assumed it always quits when the game itself quits, or crashes. I guess a persistent mode in the system tray could be handy at times.

**Comment by btbd** [comment]
@Nolram12345 @BoffinBrain The launcher currently exits after it loads the DLL into Mirror's Edge since it has no reason to stay alive after that. The message box itself is supposed to be the indicator, but a better solution would be a more complete launcher with options like disabling auto-update or whatnot.

**Comment by BoffinBrain** [comment]
I made the following batch file to speed up the launch process just a little!

```
start MMultiplayer_Launcher.exe
start steam://rungameid/17410
```
